core/src/main/scala/io/iteratee/internal/Step.scala

```diff
@@ -282,6 +282,35 @@ final object Step { self =>
   }
 
   /**
+   * A [[Step]] that sums of values in a stream.
+   *
+   * @group Collection
+   */
+  final def sum[F[_], E](implicit F: Applicative[F], M: Monoid[E]): Step[F, E, E] = new SumCont(M.empty)
```

Comment:
I wonder if combine is a better name here. Algebird used plus/sum but @non really does not like that except for explicitly"additive" (in name monoid)